pkg/auth/auth.go

```diff
@@ -307,13 +307,18 @@ func encryptUserPassword(user *models.User) error {
 // generateToken generates a JWT token for the user.
 func generateToken(user models.User) (string, error) {
 	tokenLifespan, err := strconv.Atoi(os.Getenv("TOKEN_HOUR_LIFESPAN"))
-
 	if err != nil {
 		return "", err
 	}
 
+	jwtUser := models.JWTUser{
+		Id:        user.Id,
+		Username:  user.Username,
+		Userlevel: user.Userlevel,
+	}
+
 	claims := jwt.MapClaims{}
-	claims["id"] = user.Id
+	claims["user"] = jwtUser
```

Comment:
This change will require authentication middleware to be updated as well otherwise it cannot fetch the correct user information

https://github.com/fossology/LicenseDb/blob/db6b3f31e4c206281f91e2e953829a965aad4a55/pkg/middleware/middleware.go#L82